js/script.js

```diff
@@ -0,0 +1,20 @@
+let mobileMenu = document.getElementById('hamburger-div');
+let mobileMenuCancel = document.getElementById('cancel-hamburger');
+let mobileMenuCancel1 = document.getElementById('cancel-butt1');
+let mobileMenuCancel2 = document.getElementById('cancel-butt2');
+let mobileMenuCancel3 = document.getElementById('cancel-butt3');
```

Comment:
These are not actually buttons but the links to other sections like `projects`, `contact-us` and `about`. instead of giving these `id` that doesn't describe them. why don't we just have one class name to all of them and query the class name instead?

Say we use a class name of `nav-deactivator` our code can be like

```js
const navDeactivators = document.querySelectorAll('.nav-deactivator');

const removeActiveClass = () => {
  navUl.classList.remove('active');
}

navDeactivators.forEach(navDeactivator => {
  navDeactivator.addEventListener('click', removeActiveClass);
});
```
By doing this, if we create more links in the mobile nav in the future, just adding the `nav-deactivator` class will give the link the functionality to deactivate the mobile nav